File: src/frontend/app/src/main/java/com/fiap/softtekapp/data/model/ApoioDto.kt
package com.fiap.softtekapp.data.model

import com.google.gson.annotations.SerializedName

data class ApoioDto(
    @SerializedName("codigo") val codigo: String,
    @SerializedName("titulo") val titulo: String,
    @SerializedName("descricao") val descricao: String?,
    @SerializedName("tipo") val tipo: String,
    @SerializedName("dataCriacao") val dataCriacao: String
)

File: src/backend/Softtek.Application/DTOs/ApoioDto.cs
using NUlid;

namespace Softtek.Application.DTOs
{
    public class ApoioDto
    {
        public Ulid Codigo { get; set; }
        public string Titulo { get; set; } = string.Empty;
        public string? Descricao { get; set; }
        public string Tipo { get; set; } = string.Empty;
        public DateTime DataCriacao { get; set; }
    }
}
